{"name": "Проверка увеличения счётчика \"Выполнено за все время\" и \"Выполнено за сегодня\" при создании заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: (144, 715) is out of bounds of viewport width (1064) and height (595)\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nMoveTargetOutOfBoundsError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:473:5\nassertInViewPort@chrome://remote/content/shared/webdriver/Actions.sys.mjs:2131:11\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1026:21\ndispatch/pendingEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1845:14\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1844:39\ndispatch/chainEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1770:27\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1772:7\nperformActions@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:516:23\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:156:31", "trace": "self = <tests.test_orders_feed.TestOrdersFeed object at 0x000001DCDEC45370>, driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"145c4d90-1701-4983-aa7c-5d0f5c6212ef\")>, counter_name = 'Выполнено за все время:'\n\n    @allure.title('Проверка увеличения счётчика \"Выполнено за все время\" и \"Выполнено за сегодня\" при создании заказа')\n    @allure.description('Выполняется оформление заказа и проверяется увеличение счётчика в ленте заказов.')\n    @pytest.mark.parametrize('counter_name', COUNTER_NAME)\n    def test_orders_counter(self, driver, counter_name):\n        main_page = MainPage(driver)\n        orders_feed_page = OrdersFeedPage(driver)\n        main_page.open_url(MAIN_URL)\n        main_page.set_authorization(self.user_token)\n        main_page.click_to_button_enter_account()\n        main_page.click_to_button_order_feed()\n        current_counter = int(orders_feed_page.get_counter_value(counter_name))\n>       main_page.set_order()\n\ntests\\test_orders_feed.py:61: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\main_page.py:71: in set_order\n    self.add_ingredients()\npages\\main_page.py:66: in add_ingredients\n    self.add_ingredient(ingredient)\npages\\main_page.py:61: in add_ingredient\n    self.drag_and_drop_ingredient(element, target)\npages\\base_page.py:56: in drag_and_drop_ingredient\n    ActionChains(self.driver).drag_and_drop(element, target).perform()\nvenv\\lib\\site-packages\\selenium\\webdriver\\common\\action_chains.py:94: in perform\n    self.w3c_actions.perform()\nvenv\\lib\\site-packages\\selenium\\webdriver\\common\\actions\\action_builder.py:96: in perform\n    self.driver.execute(Command.W3C_ACTIONS, enc)\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x000001DCDC1F6880>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"move target out of bounds\",\"message\":\"(144, 715) is out of bounds of view...ys.mjs:516:23\\\\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:156:31\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: (144, 715) is out of bounds of viewport width (1064) and height (595)\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nE       MoveTargetOutOfBoundsError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:473:5\nE       assertInViewPort@chrome://remote/content/shared/webdriver/Actions.sys.mjs:2131:11\nE       dispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1026:21\nE       dispatch/pendingEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1845:14\nE       dispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1844:39\nE       dispatch/chainEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1770:27\nE       dispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1772:7\nE       performActions@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:516:23\nE       receiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:156:31\n\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:229: MoveTargetOutOfBoundsException"}, "description": "Выполняется оформление заказа и проверяется увеличение счётчика в ленте заказов.", "steps": [{"name": "Открытие страницы 'https://stellarburgers.nomoreparties.site'", "status": "passed", "parameters": [{"name": "url", "value": "'https://stellarburgers.nomoreparties.site'"}], "start": 1721872660733, "stop": 1721872662328}, {"name": "Авторизация в браузере", "status": "passed", "parameters": [{"name": "argument", "value": "'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY2YTFiMGI5OWVkMjgwMDAxYjQ3YWQ2MSIsImlhdCI6MTcyMTg3MjU2OSwiZXhwIjoxNzIxODczNzY5fQ.R9Px6Rus-oNiEK0km6dSkb1ViBZS-o2KsY5eC1g5YtA'"}], "start": 1721872662328, "stop": 1721872662342}, {"name": "Клик на кнопку \"Войти в аккаунт\"", "status": "passed", "start": 1721872662342, "stop": 1721872662643}, {"name": "Клик на кнопку \"Лента заказов\"", "status": "passed", "start": 1721872662643, "stop": 1721872662891}, {"name": "Получение значения счётчика 'Выполнено за все время:'", "status": "passed", "parameters": [{"name": "counter_name", "value": "'Выполнено за все время:'"}], "start": 1721872662891, "stop": 1721872663426}, {"name": "Создание заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: (144, 715) is out of bounds of viewport width (1064) and height (595)\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nMoveTargetOutOfBoundsError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:473:5\nassertInViewPort@chrome://remote/content/shared/webdriver/Actions.sys.mjs:2131:11\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1026:21\ndispatch/pendingEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1845:14\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1844:39\ndispatch/chainEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1770:27\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1772:7\nperformActions@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:516:23\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:156:31\n\n", "trace": "  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\main_page.py\", line 71, in set_order\n    self.add_ingredients()\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\main_page.py\", line 66, in add_ingredients\n    self.add_ingredient(ingredient)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\main_page.py\", line 61, in add_ingredient\n    self.drag_and_drop_ingredient(element, target)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\base_page.py\", line 56, in drag_and_drop_ingredient\n    ActionChains(self.driver).drag_and_drop(element, target).perform()\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\common\\action_chains.py\", line 94, in perform\n    self.w3c_actions.perform()\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\common\\actions\\action_builder.py\", line 96, in perform\n    self.driver.execute(Command.W3C_ACTIONS, enc)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Клик на кнопку \"Конструктор\"", "status": "passed", "start": 1721872663426, "stop": 1721872663670}, {"name": "Добавление ингредиентов в заказ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: (144, 715) is out of bounds of viewport width (1064) and height (595)\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nMoveTargetOutOfBoundsError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:473:5\nassertInViewPort@chrome://remote/content/shared/webdriver/Actions.sys.mjs:2131:11\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1026:21\ndispatch/pendingEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1845:14\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1844:39\ndispatch/chainEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1770:27\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1772:7\nperformActions@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:516:23\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:156:31\n\n", "trace": "  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\main_page.py\", line 66, in add_ingredients\n    self.add_ingredient(ingredient)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\main_page.py\", line 61, in add_ingredient\n    self.drag_and_drop_ingredient(element, target)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\base_page.py\", line 56, in drag_and_drop_ingredient\n    ActionChains(self.driver).drag_and_drop(element, target).perform()\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\common\\action_chains.py\", line 94, in perform\n    self.w3c_actions.perform()\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\common\\actions\\action_builder.py\", line 96, in perform\n    self.driver.execute(Command.W3C_ACTIONS, enc)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Добавление ингредиента 'Флюоресцентная булка R2-D3' в заказ", "status": "passed", "parameters": [{"name": "ingredient", "value": "'Флюоресцентная булка R2-D3'"}], "start": 1721872663671, "stop": 1721872664227}, {"name": "Добавление ингредиента 'Соус Spicy-X' в заказ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: (144, 715) is out of bounds of viewport width (1064) and height (595)\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nMoveTargetOutOfBoundsError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:473:5\nassertInViewPort@chrome://remote/content/shared/webdriver/Actions.sys.mjs:2131:11\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1026:21\ndispatch/pendingEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1845:14\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1844:39\ndispatch/chainEvents<@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1770:27\ndispatch@chrome://remote/content/shared/webdriver/Actions.sys.mjs:1772:7\nperformActions@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:516:23\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:156:31\n\n", "trace": "  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\main_page.py\", line 61, in add_ingredient\n    self.drag_and_drop_ingredient(element, target)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\pages\\base_page.py\", line 56, in drag_and_drop_ingredient\n    ActionChains(self.driver).drag_and_drop(element, target).perform()\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\common\\action_chains.py\", line 94, in perform\n    self.w3c_actions.perform()\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\common\\actions\\action_builder.py\", line 96, in perform\n    self.driver.execute(Command.W3C_ACTIONS, enc)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\YAORLOVA2\\PycharmProjects\\YandexHomeWork\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "parameters": [{"name": "ingredient", "value": "'Соус Spicy-X'"}], "start": 1721872664227, "stop": 1721872664251}], "start": 1721872663671, "stop": 1721872664252}], "start": 1721872663426, "stop": 1721872664252}], "parameters": [{"name": "driver", "value": "'firefox'"}, {"name": "counter_name", "value": "'Выполнено за все время:'"}], "start": 1721872660733, "stop": 1721872664253, "uuid": "585d144c-02eb-4fbe-89f8-9e3d76ef1e8c", "historyId": "53b257a480042d495174ff9c079934a2", "testCaseId": "52e539b6eeaa4bb59cb156bf85d6924b", "fullName": "tests.test_orders_feed.TestOrdersFeed#test_orders_counter", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders_feed"}, {"name": "subSuite", "value": "TestOrdersFeed"}, {"name": "host", "value": "0000NBB01479NSR"}, {"name": "thread", "value": "22280-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders_feed"}]}